### ISSUE TITLE
Organization Update Name

### ISSUE THREAD

**Issue by jadekstewart3** [rubyforgood/pet-rescue#165]
As an organization admin, when I visit "slug/organization/edit".
-  I see a "Name" field.
- When I enter text into the "Name" field, and click "Update", I can see that the name of my organization has been updated.
- If I fail to enter text and click the "Update", I am shown an error message and redirected to the 'edit organization' page.

[Figma](https://www.figma.com/file/4PqQX8NeOVE9ca2GT4Xz76/Pet-Rescue?type=design&node-id=10-4&mode=design&t=rkMfXskGSM7Tivas-4)

**Comment by LittleBigProgramming** [comment]
Hey :) I would like to work on this if it's available?

**Comment by jadekstewart3** [comment]
@LittleBigProgramming you got it :)

**Comment by LittleBigProgramming** [comment]
@jadekstewart3 Just checking if this is ready to be worked on, can't seem to find Organizations within the dashboard. 

![image](https://github.com/rubyforgood/pet-rescue/assets/20643587/ec26f56e-2b9d-4644-bfba-bafc769a52f0)

It could be I'm missing something or this route doesn't exist and needs adding as part of this issue? 

Could you please clarify? Same for #166 :)

**Comment by jadekstewart3** [comment]
@LittleBigProgramming Ahh! 😱 I'm so sorry about that! It looks like we have some more work to do for Organizations before this will be ready. I believe we will be going over tickets during our meeting this evening, after that I should have some clarification!

**Comment by jadekstewart3** [comment]
@LittleBigProgramming Okay, so it looks like the Organization Dashboard route does not exist yet. I will create a new issue for the creation of this route, and if you would like, you can take it :) Sorry about all of the confusion, thank you for taking on issues for us!
Let me know if you have anymore questions :)

**Comment by LittleBigProgramming** [comment]
That makes sense! Thank you for looking into this :) 

If it's still available I will pick up that issue and unblock this and #166 in the process :+1:

**Comment by jadekstewart3** [comment]
@LittleBigProgramming I looked into this further yesterday, and the Organization Dashboard did exist, it was being routed through the index action on the Organization dashboard controller. Sense it was a small fix, I went ahead and did it to unblock these issues for ya :) Now we should be able to begin building out the edit org functionality!

**Comment by LittleBigProgramming** [comment]
> @LittleBigProgramming I looked into this further yesterday, and the Organization Dashboard did exist, it was being routed through the index action on the Organization dashboard controller. Sense it was a small fix, I went ahead and did it to unblock these issues for ya :) Now we should be able to begin building out the edit org functionality!

Oh fantastic! I'll pull the changes and get started

**Comment by LittleBigProgramming** [comment]
Hi, 

Sorry can you please clarify which URL needs to be changed and where it's navigated from?  

In the issue it says `slug/organization/edit` which I can't seem to route too. The `organization_profile/edit` however I can route too, which I am assuming is editing the current organization. 

However the screen shown:

![image](https://github.com/rubyforgood/pet-rescue/assets/20643587/2ecef40c-ac87-4e5b-ae70-00bc32110104)

does not seem to match the designs which have the URL `/:organization/profile/edit` seems to look quite different.  

![image](https://github.com/rubyforgood/pet-rescue/assets/20643587/356a6cf0-190d-488a-9288-7f06bb970335)

Could you please explain which url the form should be changed on and if the design within this ticket is what it is expected to look like? 

Thanks for the explanation, I'm just not sure if I'm confusing two routes or missing some context :)

**Comment by jadekstewart3** [comment]
@LittleBigProgramming Hi! Thank you for asking questions, this helps me write better tickets! Okay so this ticket is a little bit tricky, I think,🧐 because one editable attribute lives on the `organizations` table, and the others live on the `organizations_profile` table. So here are my thoughts, and I'll tag the rest of the team below cause I'm a jr, and I need sr input lol

- I think that the url should be `organization_profile/edit ` because if I understand how delegate works, you *should* be able to delegate the name attribute to the `organization_profile` and have that attribute available to you.
- I also believe that we do want to you use the geeks ui template for styling of this page, so it will require some view page work.
@kasugaijin @nsiwnf  @edwinthinks  What do ya'll think?

**Comment by kasugaijin** [comment]
Hi, @LittleBigProgramming you are correct, the route is `/organization_profile/edit` and the routes are set up to also update the profile. Things changed a bit since the writing of this ticket. Name is on the Organization instead of the OrganizationProfile.
I think this ticket can be closed, but you are correct in that the layout is not what we want.

I have made this ticket: https://github.com/rubyforgood/pet-rescue/issues/328 to update the UI, which will then unblock the #166 Please comment on that ticket if you want it and I will assign you :) Thanks for your patience!

**Comment by kasugaijin** [comment]
Completed.
